tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -165,6 +165,31 @@ ${builder.info.citation}
 % endif
 </%def>
 
+<%def name="display_figure(builder)">\
+<%
+  import requests
+  import os
+
+  config_name = os.path.split(builder.info.full_name)[-2]
+  ds_name_config = config_name.replace("\\", "/").replace("/", "_")
+
+  def dataset_examples_paths(ds_name):
+    github_path = "https://" + "github.com/Eshan-Agarwal/datasets/blob/patch-58/docs/catalog/images/" + ds_name + ".jpg"
```

Comment:
You should not store images in your repository, nor use request. Instead files should be accessed with tf.io.gfile.exists

Reply:
Also the path should be a constant passed in document_dataset.py

Reply:
images are stored in docs/catlog/images/ and here use this to check if file exists to show some results, I will correct this

Reply:
I'm talking about: `github_path = "https://" + "github.com/Eshan-Agarwal/datasets/blob/patch-58/docs/catalog/images/`

You are hardcoding your own repo url, then use `request` to query the images. Instead you should use the `tf.io.gfile` API